def.sw

```diff
@@ -0,0 +1 @@
+def xs : Int = 3 end
```

Comment:
Oops, no, just a stray file from some testing.  I have been using [jujutsu](https://martinvonz.github.io/jj/latest/) which is superior to `git` in almost every way, except that it is easier to accidentally include stray changes like this (since it automatically tracks all changes by default, including the creation of new files).

Reply:
@byorgey that `jj` seems dangerous but very tempting. 😆

Reply:
For the most part, I have found it the opposite of dangerous.  For example there is `jj undo`!  And if you really get things wedged you can easily roll back to any previous point in the history of the repository (not just any previous *commit*; any previous *state of the repository*!).  I have used this feature a few times.